docs/guides/snowflake_integration.md

```diff
@@ -486,3 +505,128 @@ FN_T_UID2_IDENTITY_MAP(EMAIL_HASH, 'email_hash')
 When you have the new function implemented, you can check the `UNMAPPED` column returned by the `FN_T_UID2_IDENTITY_MAP`. If any DII could not be mapped to a UID2, this column gives the reason.
 
 For details about the values and their explanations, see [Values for the UNMAPPED Column](#values-for-the-unmapped-column).
+
+## Usage for UID2 Sharers
+
+A UID2 sharer is any participant that wants to share UID2s with another participant. <!-- For details, see [UID2 Sharing Overview](../sharing/sharing-overview). -->
+
+A sharing participant must encrypt [Raw UID2s](../ref-info/glossary-uid#gl-raw-uid2) into [UID2 tokens](../ref-info/glossary-uid#gl-uid2-token) before sending them to another participant.
+
+The following functions support UID2 sharing:
+
+- [FN_UID2_ENCRYPT](#fn_uid2_encrypt)
+- [FN_UID2_DECRYPT](#fn_uid2_decrypt)
+
+### FN_UID2_ENCRYPT
+
+This function encrypts a raw UID2 and returns the corresponding UID2 token.
+
+Use the applicable prefix to indicate your role:
+- For advertisers: `ADV.FN_UID2_ENCRYPT`
+- For data providers: `DP.FN_UID2_ENCRYPT`
+
+>NOTE: The raw UID2 value is a Base64-encoded string, 44 characters in length.
+
+#### FN_UID2_ENCRYPT: Syntax
+
+The following is the basic syntax for the encrypt function:
+
+```
+Function: FN_UID2_ENCRYPT(raw_uid VARCHAR(44))
```

Comment:
I think we should use the same format as we have for `FN_T_UID2_IDENTITY_MAP`: a table with the arguments, data types, and description.

Same comment applies to `FN_UID2_DECRYPT`.

Reply:
AFAIK snowflake doesn't document functions this way, it does something like this
https://docs.snowflake.com/en/sql-reference/functions/floor

### Define the function 
```
FN_UID2_ENCRYPT(<raw_uid>)
```

### Arguments
<raw_uid> - varchar value of the raw UID value

### Returns 
Returns encrypted UID2 Tokens in varchar type.



But if we just wanna be quick we can just skip the varchar numbers and keep the rest like
```
Function: FN_UID2_ENCRYPT(raw_uid VARCHAR)
RETURNS VARCHAR
```

Thoughts @mcollins-ttd  @zaiweidu ?